src/main/java/org/prebid/server/vertx/verticles/server/DaemonVerticle.java

```diff
@@ -33,12 +33,12 @@ public DaemonVerticle(List<Initializable> initializables, List<ScheduledReporter
 
     @Override
     public void start(Promise<Void> startPromise) {
-        startPromise.handle(all(initializables, initializable -> initializable::initialize));
+        all(initializables, initializable -> initializable::initialize).onComplete(startPromise);
```

Comment:
is this the bug fix for https://github.com/prebid/prebid-server-java/pull/2733#issuecomment-2299529107 ?

Reply:
Yes, the primary goal of this PR is to address that comment. 
However, we need confirmation from @And1sS, as it was his idea to implement it this way.